.github/workflows/scripts/lighthouse/lighthouserc-auth.json

```diff
@@ -8,7 +8,9 @@
 				"non-composited-animations": "off",
 				"content-width": "off",
 				"csp-xss": "off",
-				"uses-passive-event-listeners": "off"
+				"uses-passive-event-listeners": "off",
+				"bf-cache": "warn",
```

Comment:
We use Spring security to configure cache header. Please turn off this rule to avoid unnecessary clutter in CI logs.